src/api/item.js

```diff
@@ -32,4 +32,16 @@ app.get('/all', (req, res) => {
     });
 });
 
+app.delete('/', (req, res) => {
+    addAction('DELETE_ITEM', {
```

Comment:
I think we ought to start normalizing the action names, so we don't have some like MODEL_CREATED and others like NEW_ITEM.
We could normalize to the [imperative][] mood (e.g. DELETE_ITEM), or the [indicative][] mood (e.g. ITEM_DELETED).

[imperative]: http://www.learnenglish.de/grammar/moodimperative.html
[indicative]: http://www.learnenglish.de/grammar/moodindicative.html